tasks/unit_tests/testdata/collector/awscontainerinsightreceiver_manifest.yaml

```diff
@@ -0,0 +1,220 @@
+dist:
```

Comment:
Suggestion: you don't have to include all the possible components for each test file. It makes it difficult to reason about what you want to test when looking at these files. Take this one for example you can just leave `awscontainerinsightreceiver` and other mandatory components then remove others.

Same for other yamls. You can leave one yaml to test all components but not necessarily all of these.

Reply:
I simplified the yamls to make it clear which is testing what. I want to make sure to test each either required and incompatible component individually, though.